### PR TITLE
add `respond_to_read` option to Switch

### PR DESCRIPTION
**Pull request by farmio**
## Description

Allows a Switch device to optionally respond to GroupValueRead requests.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

- [x] I have performed a self-review of my own code
- [x] The documentation has been adjusted accordingly
- [x] The changes generate no new warnings
- [x] Tests have been added that prove the fix is effective or that the feature works
- [x] The changes are documented in the changelog
- [ ] The Homeassistant plugin has been adjusted in case of new config options

### CODE REVIEW

**Comment by marvin-w** [comment]
That's a nice idea, do you have a use case for this?

LGTM

**Review by marvin-w — APPROVED** [review]

File: docs/switch.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Switches are simple representations of binary actors. They mainly support switch
 - `name` is the name of the object.
 - `group_address` is the KNX group address of the switch device. Used for sending.
 - `group_address_state` is the KNX group address of the switch state. Used for updating and reading state.
+- `respond` if `True` GroupValueRead requests to the `group_address` are answered. Defaults to `False`
```
Comment (marvin-w) [inline docs/switch.md:20]: not sure if I like the naming of this, but cant think of something better^^

Comment (farmio) [reply docs/switch.md:20]: I was thinking of 
- `respond_to_bus` little bit to long and where else would we respond to
- `read_flag` like ETS - but xknx is device level, not communication object
- `manage_state` but we are not really "managing" anything, just responding
- `react_to_read` / `respond_to_read` ¯\_(ツ)_/¯ 
so I settled at `respond`. The RemoteValue method for actually responding is called the same - which is a bit unfortunate. If you like one better we can change it.

**Comment by farmio** [comment]
I use similar functionalities in other KNX logic servers. E.g. for enable/disable heating (like a main switch overriding automatics); disable specific light modes that would be automatically triggered; enable/disable warm water extra heating etc.
Mostly things used in multiple knx devices or are part of logic blocks that I want to control manually.

File: docs/switch.md (hunk 1 above)
Comment (marvin-w) [reply docs/switch.md:20]: `respond_to_read` or `answer_read_request` would be my favorites. But I'm also fine if we leave it, it's just that "respond" could mean anything and I would personally prefer to have it a little bit more distinct at the cost of a potential longer name

**Pull request merged**